spec/models/ingredient_spec.rb

```diff
@@ -20,4 +20,16 @@
       end
     end
   end
+
+  describe 'has many' do
```

Comment:
You should follow exactly the same describe context it pattern as you have used in the first model spec file. 'when observing relationships' 'with movies' etx